resources/db/patches/mysql/0303~release_naming_regexes.sql

```diff
@@ -0,0 +1,16 @@
+INSERT INTO release_naming_regexes (id, group_regex, regex, status, description, ordinal)
+VALUES (
+    1161,
+    '^alt\\.binaries\\.(anime|multimedia.anime)\\.?(highspeed|repost)?',
+    '/^[[(]\\d{1,}\\/\\d{1,}[])] - "(?P<match0>.+?)[. ](vol|par).+ yEnc$/',
```

Comment:
I don't believe there's any errors, if you're taking the regex as it is and testing it, it will fail because all of the backslashes have to be escaped so in sql it looks like: /^[[(]\\d{1,}\\/\\d{1,}[])] - "(?P<match0>.+?)[. ](vol|par).+ yEnc$/
However when imported the regex will actually look like this: /^[[(]\d{1,}\/\d{1,}[])] - "(?P<match0>.+?)[. ](vol|par).+ yEnc$/